### DISCUSSION TITLE
关于 WEB 官网

### DISCUSSION THREAD

**Discussion by nick-cjyx9** [open-ani/animeko#293] [Ideas]
如果你有兴趣参与贡献，可以从这里开始：

https://github.com/open-ani/ani-website/issues/5

**Comment by Him188** [comment]
好好好


CF pages可能仍然会有连通性问题，例如移动

你可以直接写一个 docker 容器出来，ani 有服务器跑（目前在香港），也不需要刻意省成本

**Reply by nick-cjyx9** [reply]
好好好，刚看了下 Astro 造 docker 还挺简单的，会做的。
刚跑了下 cfpage 的连通性，没想到确实西北全红。

**Comment by nick-cjyx9** [comment]
**大家可以贡献一点 F.A.Q. 条目**，感觉 ANi 要扩充自己弹幕服务器的弹幕量不能只靠会折腾的这部分用户，~~真的不会在注册 bgm 这 part 卡掉一堆萌新吗~~

**Reply by Him188** [reply]
在扩了，发弹幕实际上是少数人，需要有大量用户才能有弹幕量。话说目前弹幕其实不需要折腾吧，是不需要账号就能获取的。

要求bgm是因为追番数据是bgm的，不太容易做离线模式

**Reply by nick-cjyx9** [reply]
但是实际上有很大一部分人觉得在电脑上装个 qbittorrent 并且配好 tracker 已经算是折腾了，需要缓冲一段时间更是直接让他们用回“樱花动漫”。（实例：我同学）

**Reply by nick-cjyx9** [reply]
要知道用 bgm 的，也是少数，感觉离线模式不是没有必要。

**Reply by Him188** [reply]
离线模式的话, 也不能发弹幕. 弹幕必须绑定到用户, 否则会被滥用. 所以登录还是必须要有的. 我们需要用户, 但同时也要保证平台质量. 有很多人都很在意弹幕质量

**Reply by Him188** [reply]
> 但是实际上有很大一部分人觉得在电脑上装个 qbittorrent 并且配好 tracker 已经算是折腾了

我知道这点, ani 的目标是尽可能减少人工操作, 也就是最终实现下载就能用. 

其实在安卓现在就是下载完 (登录) 就可以用, 新番缓冲个几秒就能看, 2020 以内的缓冲都挺快的. 自动缓存功能也能一定程度解决缓冲等待问题. 现在正在接在线视频数据源 (即各种网站), 虽然清晰度不高, 但至少懒得折腾或者不愿意等几秒加载的人可以用

**Reply by nick-cjyx9** [reply]
好好好，ani 前途一片光明啊。(
但是这方面的视频的弹幕怎么做呢。直接用弹弹play的？

**Reply by Him188** [reply]
而且在线源是不需要 qbit 的

**Reply by Him188** [reply]
弹幕目前是弹弹 play + ani 自己服务器

现在 AI 弹幕过滤模型快好了, 到时候会在 ani 服务器做云过滤. 跑模型很消耗算力, 应该会考虑把弹弹 play 的弹幕跑一遍模型然后塞 ani 数据库里, 定期更新.

**Reply by Him188** [reply]
发送弹幕是发送到 ani 服务器, 跟 bangumi user id 绑定, 未来可以做举报和屏蔽功能

**Reply by Him188** [reply]
目前依靠弹弹 play 的弹幕 (虽然它也其实是爬的别的网站的), 很多番都是几千条的, 能铺满手机全屏那个程度. 基本没遇到几个弹幕量特别少的, 所以效果还不错

**Reply by nick-cjyx9** [reply]
嗯，这样可行。等在线视频源有了就可以考虑在b站这种社区宣传一下了

**Reply by NeKoOuO** [reply]
我覺得ani要接在線的話可以用自定義源感覺不錯

**Comment by nick-cjyx9** [comment]
[开坑了](https://github.com/nick-cjyx9/ani-website)，现在还几乎处于新建文件夹阶段，欢迎 PR 。

cf deploy: https://ani-website.pages.dev/
